dlk/python/dlk/templates/src/func/average_pool.cpp

```diff
@@ -32,30 +32,38 @@ void func_AveragePool(const TensorView<T_FLOAT, MemoryLayout::NHWC>& input,
   int idx_out = 0;
```

Comment:
It looks we can remove this `idx_out`. (In the new code, `idx_out` is defined at line 58.)

Reply:
Oh, yes.
Fixed!